Makefile

```diff
@@ -11,8 +11,5 @@ test:
 release:
 	./gradlew :library:assembleRelease
 
-publish-stable:
-	./gradlew :library:bintrayUpload
-
-publish-snapshot-to-artifactory:
-	./gradlew :library:artifactoryPublish
+publish:
+	./gradlew :library:mapboxSDKRegistryUpload
```

Comment:
this script will only upload artifact to S3 but not actually creating publishing PRs. We should probably add another task to publish it.

Reply:
Good catch, thanks, will add publishing call to the release flow